config/routes.rb

```diff
@@ -267,6 +270,12 @@
   #   action: 'search_by_name',
   #   as: 'entity_name_search'
 
+
+  # Tags
+
+  resources :tags, only: [:edit, :create, :update, :destroy]
```

Comment:
btw, i know this is super-pedantic and *not* at all urgent, but at some point when we've got nothing better to do, it might be nice to sort the routes somehow -- for example alphabetically by resource name or sosomething? when i was poking around this file to try to figure out where i wanted to insert my own version of the `tags` routes, and wanted to look at other routes i knew about as examples, it was a bit hard to find my way around. i pretty quickly defaulted to `C-s` (which worked just fine!), but might it might have been nice to scroll and know there was some sort of human-defined order? again -- *super* not important, but the kind of thought that is only likely to occur to me the first time I try to find my way around the file, so thought I'd record it here.

Reply:
oh yeah, totally, routes.rb is a mess